prow/config/config.go

```diff
@@ -496,17 +496,22 @@ func (p Plank) GetDefaultDecorationConfigs(repo string) *prowapi.DecorationConfi
 	return def
 }
 
+// GetJobURLPrefix gets the job url prefix from the config
+// for the given refs. As we're deprecating the "gcs/" suffix
+// (to allow using multiple storageProviders within a repo)
+// we always trim the suffix here. Thus, every caller can assume
+// the job url prefix does not have a storageProvider suffix.
 func (p Plank) GetJobURLPrefix(refs *prowapi.Refs) string {
 	if refs == nil {
-		return p.JobURLPrefixConfig["*"]
+		return strings.TrimSuffix(p.JobURLPrefixConfig["*"], "gcs/")
```

Comment:
Somewhere in `prow/config.Load()` we need to check if such a prefix is present and if yes, emit a warning

Reply:
Done